registration-processor-default.properties

```diff
@@ -985,10 +986,13 @@ mosip.role.registration.getPostsync=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,RE
 mosip.role.registration.getPostsyncv2=REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,REGISTRATION_OFFICER,REGISTRATION_SUPERVISOR,RESIDENT
 auth.server.admin.allowed.audience=mosip-regproc-client,mosip-admin-client,mosip-resident-client,mosip-reg-client
 mosip.regproc.cbeff-validation.mandatory.modalities=Right,Left,Left RingFinger,Left LittleFinger,Right RingFinger,Left Thumb,Left IndexFinger,Right IndexFinger,Right LittleFinger,Right MiddleFinger,Left MiddleFinger,Right Thumb,EXCEPTION_PHOTO
-landing.zone.account.name=landing-zone
-landing.zone.type=DMZServer
+mosip.regproc.landing.zone.account.name=landing-zone
+mosip.regproc.landing.zone.type=DMZServer
 mosip.regproc.landing.zone.fixed.delay.millisecs=86400000
 mosip.regproc.landing.zone.inital.delay.millisecs=300000
+mosip.regproc.landing.zone.account.name=landing-zone
+mosip.regproc.landing.zone.type=DMZServer
```

Comment:
Please remove the new properties in line 993 and 994

Reply:
dene